solr/config/managed-schema

```diff
@@ -248,13 +248,28 @@
     <fieldType name="whole_strings" class="solr.TextField" omitNorms="true" sortMissingLast="true" multiValued="true">
       <analyzer>
         <tokenizer class="solr.KeywordTokenizerFactory"/>
+        <filter class="solr.LowerCaseFilterFactory"/>
       </analyzer>
     </fieldType>
 
     <!-- The StrField type is not analyzed, but indexed/stored verbatim. -->
     <fieldType name="string" class="solr.StrField" sortMissingLast="true" docValues="true" />
     <fieldType name="strings" class="solr.StrField" sortMissingLast="true" multiValued="true" docValues="true" />
 
+    <!-- The TextField type can be analyzed and indexed/stored. -->
+    <fieldType name="string_ci" class="solr.TextField" sortMissingLast="true">
+      <analyzer>
+        <tokenizer class="solr.KeywordTokenizerFactory" />
+        <filter class="solr.LowerCaseFilterFactory" />
+      </analyzer>
+    </fieldType>
+    <fieldType name="strings_ci" class="solr.TextField" sortMissingLast="true" multiValued="true">
```

Comment:
Besides omitting norms, what difference is this from the changes on line 251 to the `whole_strings` field type?

Reply:
The approach here is the minimum changes possible.

I'm matching the properties that are on `string` and `strings` where the *only* change is to use a lowercase string. It is just unfortunate that SOLR doesn't allow us to pass a case insensitive filter directly to `string` and `strings`.

The ommitting `norms="true"` is entirely the reason because that is different behavior not present.

I believe that I also noted that I am unable to preserve `docValues="true"` with this change. This is another undesired change but forced changed.